packages/quantic/force-app/main/default/lwc/quanticCategoryFacet/quanticCategoryFacet.html

```diff
@@ -3,15 +3,16 @@
     <c-quantic-placeholder variant="card" number-of-rows={numberOfValues}></c-quantic-placeholder>
   </template>
   <template if:true={hasParentsOrValues}>
-    <c-quantic-card-container title={label}>
+    <c-quantic-card-container title={label} toggle-visibility={toggleVisibility}>
```

Comment:
Hummm I don't think we should be adding the collapse function call to the quantic-card-container, mainly because the `isCollapsed` is the responsibility of the container of the quanticCardContainer and not the container itself.

I think it's doable to add an event `headerkeydown` for example that the card container can emit, and the facets can listen to that event to collapse.

That or we change the structure of the slot inside the card container and instead of having only a slot for actions, we have a slot for the header instead?

I think I prefer the first option.

Reply:
Yeah, the first approach is neat!  Will try implement the first solution!